packages/flutter/test/material/dropdown_test.dart

```diff
@@ -337,6 +339,25 @@ void main() {
     });
   }
 
+  testWidgets('Arrow icon aligns with the edge of button when expanded', (WidgetTester tester) async {
+    final Key buttonKey = new UniqueKey();
+
+    Widget build() =>
+        buildFrame(buttonKey: buttonKey, value: 'two', isExpanded: true);
```

Comment:
might be easier to read if this was all on one line, or if the line breaks were in the arguments rather than after the `=>`.